components/form-vue/group/group.ts

```diff
@@ -23,7 +23,12 @@ export default class AppFormGroup extends Vue {
 		const name = this.name;
 
 		if (!this.label) {
-			return titleCase(name);
+			return titleCase(name, {
```

Comment:
Why is this here?

Reply:
string utils title case was used specifically for group, which made no sense. I made the default behaviour of that function act more like a real title case with options to override it. these options will produce the same output group expected before that change.